src/core/include/iDynTree/Core/Rotation.h

```diff
@@ -373,6 +386,14 @@ namespace iDynTree
          */
         static Matrix3x3 RPYRightTrivializedDerivativeInverse(const double roll, const double pitch, const double yaw);
 
+        /**
+         * Return the inverse of the rate of change of right-trivialized derivative of the RPY function.
```

Comment:
Not sure this comment is accurate. I guess this method computes the rate-of-change of the inverse map between angular velocity and RPY derivative :thinking:

Reply:
`RPYRightTrivializedDerivativeInverseRateOfChange` evaluate the rate-of-change of the `RPYRightTrivializedDerivativeInverse` matrix. So yes, the comment should be:
```
 Return the rate of change of the inverse of the right-trivialized derivative of the RPY function.
```

Reply:
Fixed in 4c56a69